lib/tasks/update_received_quantity_to_package.rake

```diff
@@ -0,0 +1,15 @@
+# rake goodcity:update_received_quantity_to_package
+namespace :goodcity do
+  desc 'Update received quantities to package'
+  task update_received_quantity_to_package: :environment do
+    counter = 0
+    quantity_packages = Package.where("packages.received_quantity > 1")
+    quantity_packages.each do |package|
+      all_packages = Package.where("packages.inventory_number LIKE ?",
+        "#{package.inventory_number}%")
+      child_packages = all_packages - [package] if all_packages.count > 1
+      child_packages.any? && package.update(received_quantity: package.quantity) && counter+=1
```

Comment:
What if that package is designated? In that scenario, quantity will be 0 and we will never have received quantity as 0.

Reply:
also, if `all_packages.count == 0` then `child_packages` will not be initialized.